A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IRSStatus.js

```diff
@@ -1,7 +1,16 @@
 class CDUIRSStatus {
     static ShowPage(mcdu, index) {
         mcdu.clearDisplay()
-        let currPos = new LatLong(SimVar.GetSimVarValue("GPS POSITION LAT", "degree latitude"), SimVar.GetSimVarValue("GPS POSITION LON", "degree longitude")).toShortDegreeString();
+        mcdu.refreshPageCallback = () => {
+            CDUIRSStatus.ShowPage(mcdu, index);
+        }
+        SimVar.SetSimVarValue("L:FMC_UPDATE_CURRENT_PAGE", "number", 1);
+        let currPos = new LatLong(SimVar.GetSimVarValue("GPS POSITION LAT", "degree latitude"),
+                                  SimVar.GetSimVarValue("GPS POSITION LON", "degree longitude")).toShortDegreeString();
+        let currPosSplit = currPos.split("N")
```

Comment:
Same as above.

Reply:
Oops, correct! I'll fix that later today!

Reply:
Resolved